support-services/src/main/scala/com/gu/support/redemption/DynamoTableAsync.scala

```diff
@@ -0,0 +1,86 @@
+package com.gu.support.redemption
+
+import java.util.concurrent.CompletionException
+
+import com.typesafe.scalalogging.LazyLogging
+import software.amazon.awssdk.auth.credentials.{AwsCredentialsProviderChain, InstanceProfileCredentialsProvider, ProfileCredentialsProvider}
+import software.amazon.awssdk.regions.Region
+import software.amazon.awssdk.services.dynamodb.DynamoDbAsyncClient
+import software.amazon.awssdk.services.dynamodb.model.{AttributeValue, GetItemRequest, UpdateItemRequest}
+
+import scala.collection.JavaConverters._
+import scala.compat.java8.FutureConverters._
+import scala.concurrent.{ExecutionContext, Future}
+
+object DynamoTableAsync {
+
+  val ProfileName = "membership"
+
+  lazy val CredentialsProvider =  AwsCredentialsProviderChain.builder.credentialsProviders(
```

Comment:
I think you should be able to use the CredentialsProvider here:
https://github.com/guardian/support-frontend/blob/5a66eeae1ce6bb0b0bd1ef4bef64fde72492572b/support-services/src/main/scala/com/gu/aws/aws.scala#L9

Reply:
I tried originally but that's the AWS SDK v1 version, so it's in a different package although has pretty much the same name and functionality.